Allwclean

```diff
@@ -6,6 +6,11 @@ if [[ -z "${WM_PROJECT}" ]]
 then
     echo "Please source the OpenFOAM/FOAM bashrc first!"
     exit 1
+else
+    allowed_ver=("4.1" "9" "v2012" "v2016" "v2112" "v2206" "v2212" "v2306")
```

Comment:
For now (until https://github.com/solids4foam/solids4foam/pull/60 is merged), I guess we should add only 4.1, 9 and v2012.

Reply:
Oh, I forgot it's still not merged.